src/transformers/models/vivit/modeling_vivit.py

```diff
@@ -0,0 +1,746 @@
+# coding=utf-8
+# Copyright 2022 Anurag Arnab, Mostafa Dehghani, Georg Heigold, Chen Sun, Mario Lučić, Cordelia Schmid The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" PyTorch Vivit model."""
+
+
+import math
+import os
+from typing import Optional, Tuple, Union
+
+import torch
+import torch.utils.checkpoint
+from torch import nn
+from torch.nn import CrossEntropyLoss, MSELoss
+
+from ...activations import ACT2FN
+from ...modeling_outputs import BaseModelOutput, BaseModelOutputWithPooling, ImageClassifierOutput
+from ...modeling_utils import PreTrainedModel
+from ...utils import add_start_docstrings, add_start_docstrings_to_model_forward, logging, replace_return_docstrings
+from .configuration_vivit import VivitConfig
+
+
+logger = logging.get_logger(__name__)
+
+_CHECKPOINT_FOR_DOC = "jegormeister/vivit-b-16x2-kinetics400"
+_CONFIG_FOR_DOC = "VivitConfig"
+_TOKENIZER_FOR_DOC = "VivitTokenizer"
+
+VIVIT_PRETRAINED_MODEL_ARCHIVE_LIST = [
+    "jegormeister/vivit-b-16x2-kinetics400",
+    # See all Vivit models at https://huggingface.co/models?filter=vivit
+]
+
+
+def load_tf_weights_in_vivit(model, config, tf_checkpoint_path):
```

Comment:
This function can be deleted, unless you're using it (but I guess it's not used to convert weights)